api/v1/helpers.go

```diff
@@ -189,17 +189,17 @@ func (v *VerticaDB) GetPVSubPath(subPath string) string {
 
 // GetDBDataPath get the data path for the current database
 func (v *VerticaDB) GetDBDataPath() string {
-	return fmt.Sprintf("%s/%s", v.Spec.Local.DataPath, v.Spec.DBName)
+	return fmt.Sprintf("%s/%s", strings.TrimSuffix(v.Spec.Local.DataPath, "/"), v.Spec.DBName)
```

Comment:
Have we considered Go `path` [lib](https://pkg.go.dev/path/filepath#Join)?

Reply:
`strings` is fine in this case and there is not a corresponding method for this in `path/filepath`.